perun-cli/updateForm

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+#use Data::Dumper;
+
+sub help {
+	return qq{
+	Updates application form. Vo id or Group id is required.
+	--------------------------------------
+	Available options:
+	--voId           | -v vo id
+	--groupId        | -g group id
+	--autApproval    | -a automatic approval
+	--autApprovalExt | -e automatic approval extension
+	--moduleName     | -n name of module
+	--batch          | -b batch
+	--help           | -h prints this help
+
+	};
+}
+
+our $batch;
+my ($voId, $groupId, $moduleName, $app, $appE);
+GetOptions ("help|h" => sub {
+		print help();
+		exit 0;
+	},
+	"batch|b"          => \$batch,
+	"voId|g=i"         => \$voId,
+	"groupId|g=i"      => \$groupId,
+	"autApproval|a"    => \$app,
```

Comment:
We must read it as a string, otherwise param presence is considered as empty string = true boolean value. Preventing us to switch approval mode to manual (false) ``"autApproval|a=s"`` does the trick. Same goes for ``autApprovalExt|e=s``.